src/javascript/app/common/country_base.js

```diff
@@ -18,9 +18,15 @@ const isEuCountry = () => {
     );
 };
 
+const isFrance    = () => {
+    const clients_country = Client.get('residence') || State.getResponse('website_status.clients_country');
+    return /^fr$/.test(clients_country);
+};
+
 const isIndonesia = () => State.getResponse('website_status.clients_country') === 'id';
 
 module.exports = {
     isEuCountry,
     isIndonesia,
+    isFrance,
```

Comment:
since it's being place above isIndonesia, let's also maintain the same placement in the export

Reply:
Thanks. Addressed in latest commit.